packages/vaadin-template-renderer/src/vaadin-template-renderer.js

```diff
@@ -1,19 +1,69 @@
 import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer.js';
 
+import { GridElement } from '@vaadin/vaadin-grid/src/vaadin-grid.js';
```

Comment:
Seems that importing the grid here may be a problem after all. Will likely have to use some other form of detecting instead:

This works fine because the first import will register the Lumo-themed grid:
```js
import '@vaadin/vaadin-grid/all-imports';
import '@vaadin/vaadin-template-renderer';
```

This will first import (and define) the unthemed version of grid so you end up with a grid without any Lumo styling:
```js
import '@vaadin/vaadin-template-renderer';
import '@vaadin/vaadin-grid/all-imports';
```